spot/audio/main.py

```diff
@@ -31,17 +41,21 @@ def run(self, stopper: Stop, callback: Callable[[str], None]) -> None:
         """
         self.stopper = stopper
 
-        def callback_wrapper(recognizer: sr.Recognizer, audio: sr.AudioData) -> None:
+        def callback_wrapper(recognizer: Recognizer, audio: AudioData) -> None:
             if self.stopper.flag:
                 self.stop_function(True)
                 return
 
             try:
-                text = recognizer.recognize_vosk(audio).lower()
-            except sr.UnknownValueError:
-                return
+                text = recognizer.recognize_sphinx(
+                    audio,
+                    language="cs-CZ",
```

Comment:
Language CZ why? Commands are in English, does this help to account for our accent, or?